charts/ozone/Chart.yaml

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v2
+name: ozone
+description: The official Helm chart for Apache Ozone
+type: application
+version: 0.1.0
+appVersion: "1.4.0"
```

Comment:
`appVersion` should be defined as `"${project.version}"`, which will be replaced by Maven if we add a section similar to:

https://github.com/apache/ozone/blob/b87f977ef235b7ecb2062c0196dcd0a36157d6e2/hadoop-ozone/dist/pom.xml#L155-L163

It's OK to make this change as part of the "Add CI pipeline for the chart" step later.

Reply:
If charts are not intended to be part of the regular release, this is not an issue.